addons/cargo/functions/fnc_canLoadItemIn.sqf

```diff
@@ -19,6 +19,11 @@
 params [["_item", "", [objNull,""]], "_vehicle"];
 
 if (speed _vehicle > 1 || {((getPos _vehicle) select 2) > 3}) exitWith {TRACE_1("vehicle not stable",_vehicle); false};
+// TODO: use unitIsUAV command for uav check once out of devbuild
```

Comment:
`unitIsUAV` is ready now, but I don't think it will work for what we need here
It returns true for uav vehicles, but not for the AI pilots which is what we want.
.e.g. for the UGV Stomper the driver (`"B_UAV_AI"`) returns false 
(not sure why, I think it's meant for the vehicle not the units??)

So what you have right now is fine